### PR TITLE
Use http.JsonResponse when the response is JSON

### PR DESCRIPTION
**Pull request by mpasternak**
**Warning** this is untested. I have splinter errors when running tox.

http.HttpResponse with manual json.dumps call is, well, kind of acceptable. Unless you forget to set content_type to „application/json”. But then, everything still is fine, tests pass and stuff. But then, after some time, you run the code in the production, on, say, nginx+uwsgi. This combo will attach HTML tags to your json responses, head, body, the html tag itself, thus rendering json responses as unreadable to the client.

TL;DR: Use http.JsonResponse when the response is JSON

### CODE REVIEW

**Comment by mpasternak** [comment]
(I actually created this PR in hope that TravisCI will handle the testing for me xfingersx xcrossedx)

**Comment by jpic** [comment]
Works for me ;)

Go ahead and merge when you're ready

**Comment by mpasternak** [comment]
@jpic do you want me to merge this myself, did I understand correctly?

**Comment by jpic** [comment]
@mpasternak yes, when you're ready ;)
